### PR TITLE
Updating segment variable= dev key as default overrideable with ldflags

### PR DESCRIPTION
**Pull request by mohammedzee1000**
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

File: pkg/segment/segment.go [hunk 1]
```diff
@@ -19,8 +19,8 @@ import (
 	"gopkg.in/segmentio/analytics-go.v3"
 )
 
-// Writekey will be the API key used to send data to the correct source on Segment
-var WriteKey = "R1Z79HadJIrphLoeONZy5uqOjusljSwN"
+// writekey will be the API key used to send data to the correct source on Segment. Default is the dev key
+const writeKey = "4xGV1HV7K2FtUWaoAozSBD7SNCBCJ65U"
```
[the remark this answers is not in the document]
Comment (mohammedzee1000) [reply pkg/segment/segment.go:23]: ok

[the remark this answers is not in the document]
Comment (mohammedzee1000) [reply pkg/segment/segment.go]: Updated

**Review by dharmit — APPROVED** [review]
/lgtm

**Pull request merged**